BMM.Core/Implementations/Persistence/Interfaces/IBlobCacheWrapper.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+
+namespace BMM.Core.Implementations.Persistence.Interfaces
+{
+    public interface IBlobCacheWrapper
```

Comment:
Isn't it better to just call this ICache and BlobCacheWrapper is just the implementation which decides to use BlobCache to persist the cache.

Reply:
please check how many methods are actually used